tests/test_write_preds.py

```diff
@@ -23,15 +23,15 @@ def model_forward(task, batch, predict=True):
     if task.name == "sts-b":
         logits = torch.Tensor([0.6, 0.4])
         labels = torch.Tensor([0.875, 0.6])
-        out = {"n_exs": 2, "preds": [1.0, 0.8]}
+        out = {"logits": logits, "abels": labels, "n_exs": 2, "preds": [1.0, 0.8]}
```

Comment:
Minor (because these tests are not inspecting fields in `out`): spelling of `labels` key in `out` dict.